moto/core/base_backend.py

```diff
@@ -189,12 +196,13 @@ def __init__(
         if use_boto3_regions:
             sess = Session()
             self.regions.extend(sess.get_available_regions(service_name))
-            self.regions.extend(
-                sess.get_available_regions(service_name, partition_name="aws-us-gov")
-            )
-            self.regions.extend(
-                sess.get_available_regions(service_name, partition_name="aws-cn")
-            )
+            for partition in sess.get_available_partitions():
+                if partition != "aws":
```

Comment:
Minor nitpick: can we just add all partitions? `get_available_regions("ec2")` is functionally equivalent to `get_availble_regions("ec2", partition_name="aws")`, as far as I know.

So removing the initial `self.regions.extend(sess.get_available_regions(service_name))` and this if-statement should result in the same outcome, and is a bit cleaner.

Reply:
Great suggestion; fixed.